boltzrpc/boltzrpc.proto

```diff
@@ -765,6 +765,7 @@ message WalletTransaction {
     uint32 block_height = 6;
     // swaps which are related to this transaction
     repeated TransactionSwap swaps = 7;
+    bool is_consolidation = 8;
```

Comment:
Not a fan of having `TransactionType` and introducing boolean flags here. How about a `oneof` with `repeated TransactionSwap` and a single `TransactionType` and adding `CONSOLIDATION` to `TransactionType`?